jbmc/unit/java_bytecode/java_bytecode_convert_method/convert_invoke_dynamic.cpp

```diff
@@ -240,11 +240,11 @@ SCENARIO(
 
           symbol_exprt integer_param{"primitive", java_int_type()};
           symbol_exprt ref_param{"reference",
-                                 java_type_from_string("Ljava/lang/Object;")};
+                                 *java_type_from_string("Ljava/lang/Object;")};
           // NOLINTNEXTLINE(whitespace/braces)
           symbol_exprt generic_param{
             "specalisedGeneric",
-            java_type_from_string("LDummyGeneric<Ljava/lang/Interger;>;")};
+            *java_type_from_string("LDummyGeneric<Ljava/lang/Interger;>;")};
```

Comment:
❓ Is `Interger` intended here (and below)?

Reply:
I guess it doesn't matter? db6756e5 - @thk123 ?

Reply:
I guess not...! Will have a detailed look at this test, but of course no more or less broken than it was before.